accuracy/run_accuracy.jl

```diff
@@ -1,5 +1,6 @@
 # run_accuracy.jl
 
+include("../src/SPECTrecon.jl")
 using Main.SPECTrecon
```

Comment:
```suggestion
using SPECTrecon
```
This should suffice because package is registered.
You might need to use `Revise` package in your own debugging work to avoid this kind of stuff.